nominal/cli/util/verify_connection.py

```diff
@@ -0,0 +1,27 @@
+import click
+from conjure_python_client import ConjureHTTPError
+
+from nominal.core.client import NominalClient
+
+
+def validate_token_url(token: str, base_url: str) -> None:
+    """Ensure the user sets a valid configuration before letting them import the client."""
+    docs_link = "https://docs.nominal.io/python/api-tokens"
+    status_code = 200
+    err_msg = ""
+    try:
+        NominalClient.create(base_url, token).get_user()
+    except ConjureHTTPError as err:
+        status_code = err.response.status_code
+    if status_code == 401:
+        err_msg = f"The authorization token may be invalid. Read the docs on how to get a new token: {docs_link}"
+    elif status_code == 404:
+        err_msg = "The base_url may be incorrect. Ensure the url is using the api subdomain (not the app)."
```

Comment:
What does that mean? "using the api subdomain (not the app)"

Reply:
If you copy/paste the nominal URL from the web UI for e.g., you get `app.gov.nominal.io` whereas the api url is `api.gov.nominal.io`.

How about "ensure the url begins with `api` and not `app`"?

Reply:
Yes, that's helpful; should we tell them where to find the API URL, or just say: typically, you get that URL by replacing `app` with `api`?